### PR TITLE
Added Craig Fisher's modifications

### PR DESCRIPTION
**Pull request by psavery**
This includes:

- The use of a tolerance for comparing to 0 on cell boundaries
- Small changes to some comments
- Small mathematical simplifications

Craig Fisher works at the Nanostructures Research Laboratory
in Japan.

The XtalComp tests pass.

### CODE REVIEW

**Comment by psavery** [comment]
@allisonvacanti This is from Craig's email this morning. I made just a few small modifications to his code. I changed the XTALCOMP_DEBUG macro to be undefined (as is the default), and I removed a few lines of old, commented out code (since git keeps track of old code anyways).

File: xtalcomp.cpp [hunk 1]
```diff
@@ -107,7 +109,7 @@ static inline double compAngle(const XcVector &v1,
   if (angle <= 90.0)
     return angle;
   else
-    return angle - ( (angle - 90.0)+(angle - 90.0) );
```
Comment (alliepiper) [inline xtalcomp.cpp:110]: lol

**Comment by alliepiper** [comment]
I'm not sure what the purpose of using a tolerance for checking the fractional coordinates is. Seems like it might introduce numerical instability in some corner cases and shouldn't be necessary since the comparisons are performed on a supercell with a fuzzy tolerance.

Was there a problem case that these changes solve?

**Comment by psavery** [comment]
@allisonvacanti Yeah, he attached a couple of POSCAR files in the email. I forwarded the email to your new email addresses.

**Comment by alliepiper** [comment]
Ok, sounds good. Just seemed odd at a glance, but I read the email and if it corrects his issues then I'm fine merging it in. Thanks!

**Comment by psavery** [comment]
@allisonvacanti Should I add his POSCARs as test cases?

**Comment by alliepiper** [comment]
Couldn't hurt, if you've got the time.

**Pull request merged**